newIDE/app/src/SceneEditor/index.js

```diff
@@ -1352,37 +1372,6 @@ export default class SceneEditor extends React.Component<Props, State> {
             }}
           />
         )}
-        {!!this.state.variablesEditedObject && (
```

Comment:
You also need to clean the unused `variablesEditedObject` in the state, the type and the now useless `editObjectVariables`. 
Otherwise, this will clutter the file and create fake complexity for the future developer working of the file.